include/frg/formatting.hpp

```diff
@@ -383,6 +383,15 @@ void format_object(double object, format_options fo, F &formatter) {
 	_fmt_basics::format_integer(object, fo, formatter);
 }
 
+template<typename F>
+void format_object(char object, format_options fo, F &formatter) {
+	if(fo.conversion == format_conversion::null) {
```

Comment:
I think this should be inversed for reasons I mentioned before, but we should continue that discussion